Submitted By: Ken Moffat <ken at linuxfromscratch dot org>
Date: 2007-10-26
Initial Package Version: 7.0
Origin: self
Description: The 2.6.23 headers changed a couple of definitions in
cdrom.h to refer to INT_MAX which for userspace lives in limits.h.
Two files didn't already reference this, which broke the build.

Upstream status: submitted, awaiting response.

diff -Naur dvd+rw-tools-7.0.orig/growisofs.c dvd+rw-tools-7.0/growisofs.c
--- dvd+rw-tools-7.0.orig/growisofs.c   2006-09-24 18:28:53.000000000 +0100
+++ dvd+rw-tools-7.0/growisofs.c        2007-10-25 22:51:52.000000000 +0100
@@ -560,6 +560,7 @@
 #if defined(__linux)
 
 #include <linux/types.h>
+#include <limits.h>
 #include <linux/cdrom.h>
 #include <sys/ioctl.h>
 #include <sys/prctl.h>
diff -Naur dvd+rw-tools-7.0.orig/transport.hxx dvd+rw-tools-7.0/transport.hxx
--- dvd+rw-tools-7.0.orig/transport.hxx 2006-09-23 12:29:07.000000000 +0100
+++ dvd+rw-tools-7.0/transport.hxx      2007-10-25 23:01:34.000000000 +0100
@@ -132,6 +132,7 @@
 #if defined(__linux)
 
 #include <sys/ioctl.h>
+#include <limits.h>
 #include <linux/cdrom.h>
 #include <mntent.h>
 #include <sys/wait.h>

